### PR TITLE
Fix/titan cont

### PR DESCRIPTION
**Pull request by andre-merzky**
fix continuous scheduler to work as expected when memory spec is missing from resource config

### CODE REVIEW

File: bin/radical-pilot-create-static-ve [hunk 1]
```diff
@@ -12,7 +12,7 @@ echo
 
 reqs="pymongo==2.8 python-hostlist netifaces setproctitle ntplib pyzmq"
 reqs="$reqs apache-libcloud colorama backports.ssl-match-hostname msgpack-python"
-reqs="$reqs future regex"
+reqs="$reqs future regex msgpack parse"
```
Comment (iparask) [inline bin/radical-pilot-create-static-ve]: This change is also introduced in PR #1862

File: bin/radical-pilot-create-static-ve [hunk 2]
```diff
@@ -112,11 +112,16 @@ mkdir -p "$prefix"
...
+
+VIRTENV_CMD="$(which virtualenv 2>/dev/null)"
+if test -z "$VIRTENV_CMD"
+then
+    VIRTENV_TGZ="virtualenv-1.9.tar.gz"
+    VIRTENV_TGZ_URL="https://pypi.python.org/packages/source/v/virtualenv/$VIRTENV_TGZ"
+    curl -k -L -O "$VIRTENV_TGZ_URL" 2>&1 | progress
+    tar zxmf "$VIRTENV_TGZ"
+    VIRTENV_CMD="$PYTHON virtualenv-1.9/virtualenv.py"
+fi
```
Comment (iparask) [inline bin/radical-pilot-create-static-ve]: This change is also introduced in PR #1862

File: src/radical/pilot/configs/resource_ornl.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
         "valid_roots"                 : ["/lustre/atlas/scratch"],
         "rp_version"                  : "local",
         "virtenv_mode"                : "use",
-        "virtenv"                     : "/lustre/atlas2/csc230/world-shared/ve.rp.titan.20017.10.06",
+        "virtenv"                     : "/lustre/atlas2/bip149/world-shared/ve.rp.titan.2018.05.23",
```
Comment (iparask) [inline src/radical/pilot/configs/resource_ornl.json]: This change is also introduced in PR #1862

File: src/radical/pilot/configs/resource_ornl.json [hunk 4]
```diff
@@ -76,7 +76,7 @@
         "valid_roots"                 : ["/lustre/atlas/scratch"],
         "rp_version"                  : "local",
         "virtenv_mode"                : "use",
-        "virtenv"                     : "/lustre/atlas2/csc230/world-shared/ve.rp.rhea.20017.08.07",
+        "virtenv"                     : "/lustre/atlas2/bip149/world-shared/ve.rp.titan.2018.05.23",
```
Comment (iparask) [inline src/radical/pilot/configs/resource_ornl.json]: This change is also introduced in PR #1862

File: src/radical/pilot/configs/resource_ornl.json [hunk 5]
```diff
@@ -119,7 +119,7 @@
         "valid_roots"                 : ["/lustre/atlas/scratch"],
         "rp_version"                  : "local",
         "virtenv_mode"                : "use",
-        "virtenv"                     : "/lustre/atlas2/csc230/world-shared/ve.rp.titan.20017.10.06",
+        "virtenv"                     : "/lustre/atlas2/bip149/world-shared/ve.rp.titan.2018.05.23",
```
Comment (iparask) [inline src/radical/pilot/configs/resource_ornl.json]: This change is also introduced in PR #1862

File: src/radical/pilot/configs/resource_ornl.json [hunk 6]
```diff
@@ -157,7 +157,7 @@
...
```
Comment (iparask) [inline src/radical/pilot/configs/resource_ornl.json]: This change is also introduced in PR #1862

**Comment by andre-merzky** [comment]
Well, I did this PR first, and then the second one based on this one.  But yeah, either way works...

**Comment by andre-merzky** [comment]
to put it differently - once on PR is merged, the other is small - but really different...

**Pull request merged**